src/test/java/UberEatsTests.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+
+public class UberEatsTests {
+    WebDriver driver;
+    @BeforeMethod
+    public void setup(){
+        driver = new ChromeDriver();
+        driver.get("https://www.ubereats.com/fr");
+        driver.manage().window().maximize();
+        //Close cookies
+        WebElement buttonCookies = driver.findElement(By.cssSelector("[class='bc gh gi gl bj bk bl bm bn bo bt bu ba bb']"));
```

Comment:
Why use [class='bc gh gi'] when you can use `.bc.gh.gi` ?
And the selector is too fragile, please use a better selector.